tests/commons/yml/test_load_resolve.py

```
@@ -0,0 +1,32 @@
+from sema.commons.yml import LoaderBuilder
+import pytest
+import yaml
+
+
+def test_resolve_loader():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_resolve_loader`

Add return type annotation: `None`

(ANN201)